src/shared/actions/settings.js

```diff
@@ -386,6 +377,19 @@ export function getCurrencyData(currency, withAlerts = false) {
                         ),
                     );
                 }
+            })
+            .catch((error) => {
```

Comment:
`error` is unused (not sure why `eslint` didn't pick this up before)

Reply:
Fixed in https://github.com/iotaledger/trinity-wallet/pull/2013/commits/7576877be92b149065a038a3998b6c4df3051c8f